debian/changelog

```diff
@@ -1,3 +1,9 @@
+malcontent (0.7.0-1) master; urgency=medium
```

Comment:
Should this be `0.7.0-1endless1`?

Reply:
Possibly, although we are the upstream packaging at the moment (there’s no Debian package yet) so it doesn’t really make a difference. Let’s re-assess on the next rebase.